packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButton.tsx

```diff
@@ -43,20 +44,32 @@ export const RadioButton = React.forwardRef<HTMLInputElement, RadioButtonProps>(
   ) => {
     const styles = useStyles2(getRadioButtonStyles, size, fullWidth);
 
-    return (
-      <div className={styles.radioOption}>
-        <input
-          type="radio"
-          className={styles.radio}
-          onChange={onChange}
-          onClick={onClick}
-          disabled={disabled}
-          id={id}
-          checked={active}
-          name={name}
-          aria-label={ariaLabel || description}
-          ref={ref}
-        />
+    const inputRadioButton = (
+      <input
+        type="radio"
+        className={styles.radio}
+        onChange={onChange}
+        onClick={onClick}
+        disabled={disabled}
+        id={id}
+        checked={active}
+        name={name}
+        aria-label={ariaLabel}
+        ref={ref}
+      />
+    );
+    return description ? (
+      <div className={styles.radioOption} data-testid="radio-button">
```

Comment:
let's define this selector in the e2e-selectors file here: https://github.com/grafana/grafana/blob/eledobleefe/radio-button-tooltip-68901/packages/grafana-e2e-selectors/src/selectors/components.ts

something like:
```tsx
RadioButton: {
  container: `data-testid radio-button`
}
```

then you can use it like:
```tsx
<div className={styles.radioOption} data-testid={selectors.components.RadioButton.container}>